### DISCUSSION TITLE
Way to check whether there are updates to any ports when using manifest mode?

### DISCUSSION THREAD

**Discussion by nickdademo** [microsoft/vcpkg#36700] [Q&A]
Wondering if there is a `vcpkg` command which can check all ports in the project's `vcpkg.json` and print which ones have updates?

**Comment by nickdademo** [comment]
> Wondering if there is a `vcpkg` command which can check all ports in the project's `vcpkg.json` and print which ones have updates?

I assume there isn't any easy way to do this then? :) I may try to roll my own.

**Reply by Osyotr** [reply]
Try updating baseline and running `vcpkg install --dry-run`.

**Reply by nickdademo** [reply]
Thanks! Looks like this doesn't work for ports pinned to specific versions in the manifest file however...

For example, I have `libpng` set to 1.6.40 in the manifest, but the command prints the following even though 1.6.43 is available:
```
libpng:x64-windows@1.6.40 -- C:\Users\DNDT\AppData\Local\vcpkg\registries\git-trees\1a2a7f489e57c9e55e049b7c5f0a28c8ff4464ed
```

**Reply by nickdademo** [reply]
Wanted to check back on this again - is there now an easy way to do this? 🤞
